code/hash/Hacl.Hash.Definitions.fst

```diff
@@ -19,22 +19,22 @@ open FStar.Mul
 inline_for_extraction noextract
 let m_spec (a:hash_alg) : Type0 =
   match a with
-  | MD5 | SHA1 | SHA2_224 | SHA2_256 | SHA2_384 | SHA2_512 | SHA3_256 -> unit
   | Blake2S | Blake2B -> Blake2.m_spec
+  | _ -> unit
```

Comment:
Did you see proof performance improvements with this change?
I usually like the exhaustive patterns instead of relying on wildcards as it makes it easier to track required changes when adding a new algorithm, but this is a matter of personal taste.

Reply:
I did not notice any proof improvements. No problem with making this pattern (and others below) exhaustive. Fixed in 2ce8329cd6 which is part of the subsequent PR.